src/components/common/CheckboxTree.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import ReactCheckboxTree from 'react-checkbox-tree';
+
+export default function CheckboxTree({
+    checkboxName,
+    expandedState = ['<ALL>'],
+    filterState = [],
+    nodes = [],
+    onCheckFunc = () => null,
+} = {}) {
+    const [expanded, setExpanded] = useState(expandedState);
+    const dispatch = useDispatch();
+
+    const icons = {
+        check: <span className='icon-xs bg-accent w-3 h-3 rounded-sm leading-3'>check</span>,
+        uncheck: <span className='bg-inverted w-3 h-3 rounded-sm' />,
+        halfCheck: <span className='icon-xs bg-secondary w-3 h-3 rounded-sm leading-3'>check</span>,
```

Comment:
Noticing now we have very inconsistent checkboxes on the platform, haha

Requirements (LabelledCheckbox) - html `<input />` has default styles in some places
![image](https://user-images.githubusercontent.com/31848108/151714003-ac814ddb-3c16-4fbb-9f92-1e494b557fb3.png)

Task table checkboxes - google font icons (check_box, check_box_outline_blank)
![image](https://user-images.githubusercontent.com/31848108/151714012-9222d5e3-719a-4001-9349-ab2a6a1890aa.png)

What i've implemented here is a 3rd style, trying to copy the 1st `input` ones - but hard to match exactly.

What would you like to do?

Reply:
Hmm yeah I think this is close enough. Can always tweak later to make things more consistent